samples/openapi3/client/petstore/php/OpenAPIClient-php/lib/Model/FormatTest.php

```diff
@@ -80,7 +80,7 @@ class FormatTest implements ModelInterface, ArrayAccess
       * @var string[]
       */
     protected static $openAPIFormats = [
-        'integer' => 'int32',
+        'integer' => null,
```

Comment:
Looks like 'integer', 'number' and 'string' type need to be revised. I'll do it in another PR instead if issues found.

Reply:
UPDATE: Reviewed and that's the expected behaviour as it should be storing the "format" defined in the spec